### PR TITLE
Adding content to community page

### PR DESCRIPTION
**Pull request by skinnylatte**
First draft of community page

Signed-off-by: Adrianna Tan <skinnylatte@gmail.com>

### CODE REVIEW

File: content/community.md [hunk 1]
```diff
@@ -1,7 +1,61 @@
...
+
+**Services Offered:**  
+
+* DBaaS
+* Professional Services
+* Vitess Operator for Kubernetes
+* Enterprise Support
+* .Vitess Training
+
+If you provide services in the Vitess ecosystem and would like to be listed here, [let us know](cncf-vitess-maintainers@lists.cncf.io).
```
Comment (caniszczyk) [inline content/community.md]: or send us a pull request ;)?

File: content/community.md [hunk 2]
```diff
@@ -1,7 +1,61 @@
...
+* gRPC
+* Grafana 
+* Prometheus
+* etcd
+* Consul
+* Zookeeper
+
+### Commercial Vitess Support
+
+If you need commercial assistance for open source Vitess, consider getting in touch wiht the following companies.
```
Comment (caniszczyk) [inline content/community.md]: s/wiht/with

File: content/community.md [hunk 3]
```diff
@@ -1,7 +1,61 @@
...
+
```
Comment (caniszczyk) [inline content/community.md]: listed in alphabetical order?

File: content/community.md (hunk 1 above)
Comment (skinnylatte) [reply content/community.md]: Still a WIP but yes great idea

File: content/community.md (hunk 2 above)
Comment (skinnylatte) [reply content/community.md]: Catching the typos now thanks

File: content/community.md (hunk 3 above)
Comment (skinnylatte) [reply content/community.md]: I was planning to put that in when there is more than one community listed

Comment (skinnylatte) [reply content/community.md]: but yes i have put that in :)

File: content/community.md (hunk 2 above)
Comment (skinnylatte) [reply content/community.md]: resolved with the new push

**Review by sougou — APPROVED** [review]

**Pull request merged**